//
// Student License - for use by students to meet course requirements and
// perform academic research at degree granting institutions only.  Not
// for government, commercial, or other organizational use.
//
// acdefg_T_V.h
//
// Code generation for function 'acdefg_T_V'
//

#ifndef ACDEFG_T_V_H
#define ACDEFG_T_V_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void acdefg_T_V(double V_init, double A_init, double V_wayp, double V_max,
                double V_min, double A_min, double J_max, double J_min,
                double T, creal_T t[14]);

#endif
// End of code generation (acdefg_T_V.h)
